recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set md5 = "c486211efa29b95771c74f5d8701ca4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy >=1.12
+    - scipy
+
+  run:
+    - python
+    - numpy >=1.12
+    - scipy
+
+test:
+  requires:
+    - scikit-learn >=0.18.1
+    - numpy >=1.12
+    - scipy
+
+  imports:
+    - xgboost
+    - sklearn.datasets
+    - sklearn.model_selection
+    - sklearn.metrics
+
+about:
+  home: https://github.com/dmlc/xgboost
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
Asked upstream to include a license file in the future.

xref: https://github.com/dmlc/xgboost/issues/1087